File: CSharp/DP.Domain/Samples/Strategy/TextLogger.cs
using System;

namespace DP.Domain.Samples.Strategy
{
    public class TextLogger:ILogger
    {
        public void Debug(string msg) => System.Diagnostics.Trace.WriteLine($"(Text)Debug: {msg}");
        public void Warn(string msg) => System.Diagnostics.Trace.WriteLine($"(Text)Warn: : {msg}");
        public void Error(string msg) => System.Diagnostics.Trace.WriteLine($"(Text)Error: : {msg}");
    }
}

File: Python/Samples/Strategy/TextLogger.py
from BaseLogger import BaseLogger

class TextLogger(BaseLogger):
    def debug(self,msg: str):
        print("(Text)Debug: " + msg)

    def warn(self,msg: str):
        print("(Text)Warn: " + msg)

    def error(self,msg: str):
        print("(Text)Error: " + msg)
        